<!doctype html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
    <link rel="stylesheet" href="style.css">
    <link
        href="https://fonts.googleapis.com/css?family=Nunito:300,300i,400,400i,600,600i,700,700i,800,800i&display=swap"
        rel="stylesheet">
    <script defer src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
        integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous">
    </script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
        integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous">
    </script>
    </script>
    <script defer src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
        integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous">
    </script>

    <title>Alaive</title>
</head>

<body>
    <nav class="navbar sticky-top navbar-expand-lg navbar-light px-5 shadow">
        <a class="navbar-brand" href="#">
            <img src="https://2019.igem.org/wiki/images/2/2b/T--UC_San_Diego--logo.png" width="50" height="50"
                alt="Logo of Alaive">
        </a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>

        <div class="collapse navbar-collapse" id="navbarSupportedContent">
            <ul class="navbar-nav ml-auto">
                <li class="nav-item">
                    <a class="nav-link" href="http://2019.igem.org/Team:UC_San_Diego/Description">Description</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="http://2019.igem.org/Team:UC_San_Diego/Model">Model</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="http://2019.igem.org/Team:UC_San_Diego/Software">Software</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="http://2019.igem.org/Team:UC_San_Diego/Code">Code</a>
                </li>
                <li class="nav-item dropdown">
                    <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button"
                        data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                        Wetlab
                    </a>
                    <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                        <a class="dropdown-item"
                            href="http://2019.igem.org/Team:UC_San_Diego/Wetlab_Overview">Overview</a>
                        <a class="dropdown-item"
                            href="http://2019.igem.org/Team:UC_San_Diego/Wetlab_Protocol">Protocol</a>
                    </div>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="http://2019.igem.org/Team:UC_San_Diego/Team">Team</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="http://2019.igem.org/Team:UC_San_Diego/Attribution">Attribution</a>
                </li>
            </ul>
        </div>
    </nav>

    <section class="jumbotron jumbotron-fluid hero">
        <div class="container my-5">
            <h1 class="font-weight-bolder text-light text-center mb-0">WETLAB PROTOCOL</h1>
        </div>
    </section>

    <section class="container my-5">
        <h1>Prepare Jurkat T-cells</h1>
        <h2>Growth Medium</h2>
        <p>
            Advanced RPMI 1640 (Gibco/Invitrogen) + 10% fetal bovine serum (Hyclone) + 10 mM Hepes + 100 units/ml
            penicillin + 100 μg/ml streptomycin + 5% CO2 at 37o C. Liquid
        </p>
        <h2>Thawing</h2>
        <p>
            Thaw a 1-ml aliquot of cells as quickly as possible in water bath at 37° C. Transfer cells to 10 ml warm
            media in a 30-ml flask. Mix gently. Centrifuge at 1,200 rpm for 5 minutes to pellet cells. Discard media and
            resuspend pellet gently in 12 ml warm medium. Place in incubator.
        </p>
        <h2>Cell Culture</h2>
        <p>
            Cells will be passaged at 2-day intervals or when the color of the media change from amber to light yellow.
            To split the cells, remove 6 mL of media from the flask and place it in a new 30 mL flask. Add 6 mL fresh
            media to each flask.
        </p>
        <h2>Co-culturing microglia and transfering media to Jurkat T-cells.</h2>
        <p>
            12 plates of microglia will be prepared. 3 of the plates will be co-cultured with beta amyloid, another 3
            with LPS, and another 3 with poly(I:C). Nothing will be added to the last 3 plates to serve as control.
        </p>
        <p>
            Beta amyloid coculture: Aβ1–42 will be added to each plate of microglia culture at 0.5 uM.
        </p>
        <p>
            LPS coculture: LPS will be added to each plate of microglia culture at 0.1 ug/mL.
        </p>
        <p>
            polyIC coculture: polyIC will be added to each plate of microglia culture at 10 μg/mL.
        </p>
        <p>
            After 24 hrs of coculturing, drop 100 uL of media into the Jurkat T-cell flasks (one flask for each plate of
            microglia culture).
        </p>
        <h2>Phage Display Library Screening</h2>
        <p>
            Taken and condensed from: https://www.sciencedirect.com/science/article/pii/S0003269711007378,
            https://www.neb.com/-/media/nebus/files/manuals/manuale8100.pdf?rev=ec2f1e70b3aa4887b6d9ea3877be2f45
        </p>

        <h2>Biopanning</h2>
        <p>
            A heptapeptide phage display library (Ph.D.-7 Phage Display Peptide Library Kit, New England Biolabs,
            Beverly, MA, USA) will be used for the in vitro biopanning experiments.

        </p>
        <p>
            Jurkat cells will be washed washed gently with phosphate-buffered saline (PBS) and incubated with 5 ml of
            α-MEM (minimum essential medium), containing 0.1% (w/v) bovine serum albumin (BSA), for 1 h at 37 °C under
            5% CO2. The cells will be gently washed once with 5 ml of PBS before adding the phage library. Phages (2 ×
            1011) will be added in 3 ml of α-MEM containing 0.1% BSA. Cells will be incubated with the phage for 1 h at
            37 °C while shaking at 70 rounds per minute. After the incubation, the cells will be gently washed six times
            by incubating with 5 ml of ice-cold α-MEM, containing 0.1% BSA, for 5 min. Subsequently, the cells will be
            incubated for 10 min on ice with 3 ml of 0.1 M HCl (pH 2.2) to elute cell surface-bound phage. This solution
            will be neutralized by the addition of 0.6 ml of 0.5 M Tris. The cells will be then lysed for 1 h on ice in
            3 ml of 30 mM Tris–HCl and 1 mM EDTA (pH 8.0) to recover the cell-associated phage fraction. Phages will be
            amplified as follows:


            <ol>
                <li>Inoculate 20 ml of LB medium in a 250-ml Erlenmeyer flask (do not use a 50-ml conical tube) with
                    ER2738. Incubate culture at 37°C with vigorous shaking. Carefully monitor the 20-ml culture so that
                    it does not grow beyond early-log phase (OD600 0.01–0.05).</li>
                <li>Amplify the rest of the eluate by adding the eluate to the 20-ml ER2738 culture (should be early-log
                    at this point) and incubating with vigorous shaking for 4.5 hours at 37°C. </li>
                <li>Transfer the culture to a centrifuge tube and spin for 10 minutes at 12,000 g at 4°C. Transfer the
                    supernatant to a fresh tube and re-spin (discard the pellet).</li>
                <li>Transfer the upper 80% of the supernatant to a fresh tube and add to it 1/6 volume of 20% PEG/2.5 M
                    NaCl. Allow the phage to precipitate at 4°C for at least 2 hours, preferably overnight.</li>
                <li>Spin the PEG precipitation at 12,000 g for 15 minutes at 4°C. Decant and discard the supernatant,
                    re-spin the tube briefly, and remove residual supernatant with a pipette. The phage pellet should be
                    a white finger print sized smear on the side of the tube. </li>
                <li>Suspend the pellet in 1 ml of TBS. Transfer the suspension to a microcentrifuge tube and spin at
                    maximum (14,000 rpm) for 5 minutes at 4°C to pellet residual cells. </li>
                <li>Transfer the supernatant to a fresh microcentrifuge tube and reprecipitate by adding 1/6 volume of
                    20% PEG/2.5 M NaCl. Incubate on ice for 15–60 minutes. Microcentrifuge at 14,000 rpm for 10 minutes
                    at 4°C, discard the supernatant, re-spin briefly, and remove residual supernatant with a micropipet.
                </li>
                <li>Suspend the pellet in 200 µl of TBS. Microcentrifuge for 1 minute to pellet any remaining insoluble
                    material. Transfer the supernatant to a fresh tube. This is the amplified eluate.</li>
                <li>Titer the amplified eluate as described in General M13 Methods (page 8) on LB/IPTG/Xgal plates.</li>
                <li>Count blue plaques from the titering plates in Step 19 and determine the phage titer, which should
                    be on the order of 1013–14 pfu/ml. Use this value to calculate an input volume corresponding to the
                    input titer previously described. </li>
                <li>Apply the amplified library to Jurkat cells again.</li>
            </ol>
        </p>
        <h2>DNA preparation and sequencing: </h2>
        <p>
            Sequencing will be performed with the Illumina Whole Genome Analyzer WG2 (San Diego, CA, USA). The phages
            will be eluted as described above. Steps 1-8 above will be repeated to amplify phages. The final pellet will
            be dissolved in 25 μl of Milli-Q water, and the DNA concentration will be determined by Nanodrop before
            freezing at −20 °C. Phage DNA will be amplified with the following polymerase chain reaction(PCR) primers:
        </p>
        <p>
            Forward: AAT GAT ACG GCG ACC ACC GAG ATC TAC ACT TCC TTT AGT GGT ACC TTT CTA TTC TC*A
        </p>
        <p>
            Reverse: CAA GCA GAA GAC GGC ATA CGA GAT CGG TCT ATG GGA TTT TGC TAA ACA ACT TT*C,
        </p>
        <p>
            where an asterisk (∗) indicates phosphorothioate bond. PCR primers used to amplify the phage DNA contained a
            subsequence that recognized the sequence flanking the 21-nucleotide unknown insert sequence and the adapters
            necessary for binding to the Illumina flow cell. The final product of the PCR was 160 bp long. The PCR
            protocol applied was the following: 1 ng of phage DNA was incubated with 2.625 U of high-fidelity Taq
            polymerase (Roche Diagnostics, The Netherlands), 20 pM of primers in 1× high-fidelity PCR buffer containing
            15 mM MgCl2, and amplified for 20 cycles, each consisting of an incubation for 30 s at 94 °C, 30 s at 60 °C,
            and 30 s at 72 °C. The PCR was stopped in the exponential phase to mitigate PCR-induced sequence biases. The
            final PCR product was purified with the QIAquick PCR Purification Kit (Qiagen, Valencia, CA, USA).
            Concentrations, as well as the correct length, of the PCR product were established with an Agilent 2100
            Bioanalyzer and a DNA 1000 assay. Each PCR product was applied to a single lane of an Illumina flow cell and
            subjected to solid phase amplification in the cluster station. Single end sequencing for 27 to 35 cycles (27
            cycles are sufficient, but runs were sometimes extended to 35 cycles due to requirements for samples in
            other lanes of the same flow cell) was performed with a custom sequencing primer that started exactly at the
            first position of the unknown insert sequence (ACA CTT CCT TTA GTG GTA CCT TTC TAT TCT CAC TC∗T).
        </p>
    </section>

</body>

</html>